### PR TITLE
Active-Active developer document changes

### PR DESCRIPTION
**Pull request by kamran-redis**
Changes to Active-Active documentation
* changed explanation for list concurrent update
* stream page minor command syntax corrections
* removed counter/string duality from string type
* Corrected result for HLL example

### CODE REVIEW

**Comment by banker** [comment]
I thought it looked good, @kamran-redis. But let's ask Valentino to verify. What's his GitHub username? Can you mention him here?

**Comment by kamran-redis** [comment]
@banker I do not have  Valentino's giithub ID

**Comment by bmansheim** [comment]
I pinged him.

File: content/rs/developing/crdbs/developing-hll-crdb.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Here is an example of a simple write case:
 | t2   | --- sync ---      |                   |
 | t3   |                   | PFADD hll x       |
```
Comment (valentinogeron) [inline content/rs/developing/crdbs/developing-hll-crdb.md:24]: This command as no effect because `x` already exists in the `hll`. 
Pls change it to `PFADD hll y` and revert the change of the `PFCOUNT` result

File: content/rs/developing/crdbs/developing-streams-crdb.md [hunk 2]
```diff
@@ -261,8 +261,8 @@ For example:
 | _t4_   | _— Sync —_                                      | _— Sync —_             |
 | _t5_   | `XRANGE messages - +` <br/>**→ [110-1]**                          | XRANGE messages - + <br/>**→ [110-1]** |
 | _t6_   | `XINFO GROUPS messages` <br/>**→ [g1]**                            | XINFO GROUPS messages <br/>**→ [g1]**   |
```
Comment (valentinogeron) [inline content/rs/developing/crdbs/developing-streams-crdb.md:263]: here we should fix the group name `g1` -> `group1`
